pages/profile/permission/index.js

```diff
@@ -0,0 +1,56 @@
+const page = require('../../../lib/page');
+const form = require('../../common/routers/form');
+const schema = require('./schema');
+
+module.exports = settings => {
+  const app = page({
+    root: __dirname,
+    ...settings
+  });
+
+  app.use('/', form({ schema }));
+
+  app.use('/', (req, res, next) => {
+    res.locals.static.isNamed = !!((req.profile.roles && req.profile.roles.length) || req.profile.pil || (req.profile.projects && req.profile.projects.length));
```

Comment:
I feel like this condition probably wants to be broken down into a couple of lines rather than trying to do the whole thing in one.

Also, we'll want to check that the licences held are active as well as just existing.

Also also, I'm not 100% sure of the scope of the profile at this point but we'll likely need to check if the profile's PIL and projects are associated with the current establishment. So if a user has projects or a PIL at establishment X, then they should still be removable from establishment Y. Is this the case?

Reply:
I just made sure that profile's roles, PIL and projects are scoped under establishment.Check 
https://github.com/UKHomeOffice/asl-schema/pull/119.